bundle/manifests/skupper-operator-config_v1_configmap.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  console: "true"
+  console-authentication: internal
+  console-password: "admin"
+  console-user: "admin"
+  name: skupper-site
+  router-console: "true"
+  router-mode: interior
+  service-controller: "true"
+  service-sync: "true"
+metadata:
+  name: skupper-site
```

Comment:
@grs I am also trying to introduce a standard configmap (maybe removing most of
the attributes we have right now). The idea is to meet the "full GUI" experience so that
user clicks "Install" into the Skupper Operator and the Skupper console is available.

It seems to work, but I am going to make some more tests to gather pros and cons of
using it this way and prepare something for discussion.